modules/genomic_browser/test/genomic_browserTest.php

```diff
@@ -144,11 +94,14 @@ function testGenomicBrowserWithPermission()
     function testGenomicBrowserEachTab()
     {
         $this->_testPageUIs("/genomic_browser/", $this->_loadingProfilesUI);
-        $this->_testPageUIs("/genomic_browser/gwas_browser/", $this->_loadingGWASUI);
-        $this->_testPageUIs("/genomic_browser/snp_browser/", $this->_loadingSNPUI);
-        $this->_testPageUIs("/genomic_browser/cpg_browser/", $this->_MethylationUI);
+        $this->_testPageUIs("/genomic_browser/#tabGWAS", $this->_loadingGWASUI);
```

Comment:
I don't think it's standard to route endpoints using `#endpoint`. I don't have a problem with the change itself but I think we should make an effort to be consistent across LORIS.

If we want to do this for all modules (again not opposed to it), it would be good to have a discussion about it and document it somewhere.

Reply:
@johnsaigle that's the link to access the GWAS tab. I was updating with the test code. Side noteL I'm not sure why the test code works sometimes and doesn't otherwise.

Reply:
Got it, I thought you were introducing something new.